themes/taranis/layouts/partials/index_profile.html

```diff
@@ -3,8 +3,31 @@
     <div class="profile_inner">
         <h1>{{ .title | default site.Title | markdownify }}</h1>
         <span>{{ .subtitle | markdownify }}</span>
-        {{- if .imageUrl -}}
-        <img draggable="false" src="{{ .imageUrl | absURL }}" alt="{{ .imageTitle | default "profile image" }}" title="{{ .imageTitle }}" class="profile-banner"/>
```

Comment:
Please add this as a fallback behaviour, logic should be:

if videoUrl and imageUrl => show video with poster
if videoUrl and NOT imageUrl => show video without setting poster
if NOT videoURL and imageUrl => old behaviour aka only image
if NOT video and NOT image => show nothing

Reply:
thanks, added it.